military-symbology-styles/test-data/app6b/truth-data/App6bTruthData-LinesOnly.csv

```diff
@@ -137,13 +137,14 @@ OID,Annex,Affiliation,Full SIDC,SIDC By Parts,Hierarchy Code,Name,Geometry,Notes
 136,E,Friend,GFCPSLRW-------,G F C P SL RW -- -- -- -,2.X.2.4.2.2.5,TWO-WAY TRAFFIC,LINE,
 137,E,Friend,GFCPOLN--------,G F C P OL N- -- -- -- -,2.X.2.5.3.1,LIGHT LINE,LINE,
 138,E,Friend,GFCPOLP--------,G F C P OL P- -- -- -- -,2.X.2.5.3.2,PHASE LINE,LINE,
```

Comment:
@csmoore @BobBooth Just confirming the order an item shows up in the list doesn't affect anything else?  I suspect the answer is no, just making sure.

Reply:
@ACueva - order of items shouldn't matter - but they will appear in this same order in the output feature class when run through the [test generation process](https://github.com/Esri/military-features-data/blob/dev/military-overlay/utils-source-and-test-data/comprehensive-test-generation/README.md)